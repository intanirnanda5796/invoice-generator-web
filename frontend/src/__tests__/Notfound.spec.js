import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../pages/notfound';
import { BrowserRouter } from 'react-router-dom';

describe('NotFound', () => {
    test('snapshot renderer not found page', () => {
        const component = renderer.create(<BrowserRouter><NotFound /></BrowserRouter>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})