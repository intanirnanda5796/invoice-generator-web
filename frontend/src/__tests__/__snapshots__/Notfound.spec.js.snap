// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot renderer not found page 1`] = `
<div
  className="my-5 pt-5"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12"
      >
        <div
          className="text-center my-5"
        >
          <h1
            className="fw-bold text-error"
          >
            4
             
            <span
              className="error-text"
            >
              0
              <img
                alt=""
                className="error-img"
                src=""
              />
            </span>
             
            4
          </h1>
          <h3
            className="text-uppercase"
          >
            Sorry, page not found
          </h3>
          <div
            className="mt-5 text-center"
          >
            <a
              className="btn btn-primary waves-effect waves-light"
              href="/"
              onClick={[Function]}
            >
              Back to Dashboard
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
